### ISSUE TITLE
Problem with sass:build task

### ISSUE THREAD

**Issue by davidtruchet** [aesinv/simple-aem-fe-example#1]
There is a sync issue when you run the sass task. After you run the sass task you end up having an old version of the main.css file, that's due the callback function is been called before the task "sass:build" is done.
I was able to fix that adding this line after the last pipe call of that task:

```
gulp.task('sass:build', function (cb) {
    gulp.src(mainCss)
        .pipe(plumber()) // Prevents pipe breaking due to error (for watch task)
        .pipe(sourcemaps.init())
        .pipe(sass({
            outputStyle: 'compressed',
            omitSourceMapUrl: true, // This is hardcoded in the main.scss due to resource path issues
            includePaths: [sassPath, components]
        }).on('error', sass.logError))
        .pipe(sourcemaps.write('./', {
            addComment: false
        }))
        .pipe(plumber.stop())
        .pipe(gulp.dest(cssPath))
        .on('end', cb);
});
```

If you want I can submit a PR with the fix.

Thanks,
David.

**Comment by bengsfort** [comment]
Hey David!

Definitely, PR that sucker and I'll merge it in ASAP! Thanks a ton!

Matt

**Comment by davidtruchet** [comment]
Submitted! :)

**Comment by bengsfort** [comment]
Merged #2, thanks @davidtruchet !
